docs/tests/t007.md

```diff
@@ -0,0 +1,106 @@
+| Type | Arm | | 
+| Subject Assignment | Optional |  
+| *Study Groups* | *Name* | *Description* |
+| 1 | T007_A_Name_1 | T007_A_Description_1 |  
+| 2 | T007_A_Name_2 | T007_A_Description_2 |  
+
+**expected results:**  
+After clicking **Confirm Subject Group Class** and **Submit Subject Group Class** the message **The Subject Group Class was created successfully.** appears.
+
+## T007-02: search a subject group class in the list of groups  
+**description:**  
+This test focuses on finding a subject group class in the list of groups.
+
+**prerequisite:**  
+You must be logged in to a study for which you have the data-manager-role and you must have successfully completed step **T007-01: create a new group class**.  
+
+**workflow:**  
+1. click on **Tasks - Study Setup - Build Study**
+1. click on the magnifier-icon of task **4 Create Subject Group Classes**
+1. in the screen **Manage All Groups in Study [Study Name]** enter the name of event you created in **T007-01: create a new group class**
+1. click **Find**
+
+**expected results:**  
+The event appears in the list of results.  
+
+## T007-03: edit a subject group class
+**description:**  
+This test focuses on editing the properties of a subject group class and the underlying groups.
+
+**prerequisite:**  
+You must be logged in to a study for which you have the data-manager-role and you must have successfully completed step **T007-01: create a new group class**.  
+
+**workflow:**  
+1. click on **Tasks - Study Setup - Build Study**
+1. click on the magnifier-icon of task **4 Create Subject Group Classes**
+1. in the screen **Manage All Groups in Study [Study Name]** enter the name of event you created in **T007-01: create a new group class**
+1. click **Find**
+1. in the list of result, click on the pencil / edit icon
+1. in the screen **Update a Subject Group Class** change class name, study group names and study group descriptions by inserting **XX_** after **T007_**
+1. change user type to **Family/Pedigree** and change subject assignment to **Required**
+1. click **Confirm Subject Group Class**
+1. verify that the changes are correctly displayed in the screen **Confirm a Subject Group Class**
+1. click **Submit Subject Group Class**  
+
+**expected results:**  
+The message **The Subject Group Class was updated successfully.** appears.
+
+## T007-04: remove a subject group class
+**description:**  
+This test focuses on removing a subject group class and the underlying groups.
+
+**prerequisite:**  
+You must be logged in to a study for which you have the data-manager-role and you must have successfully completed step **T007-01: create a new group class**.  
+
+**workflow:**    
+1. click on **Tasks - Study Setup - Build Study**
+1. click on the magnifier-icon of task **4 Create Subject Group Classes**
+1. in the screen **Manage All Groups in Study [Study Name]** enter the name of event you removed in **T007-01: create a new group classs**

```

Comment:
Do you mean "... enter the name of the event you created ..."?

Reply:
You're right. I corrected it and committed a new version.

Reply:
I can't find the commit. Did you pushed the changes?

Reply:
I commited a new version just now, Nov 18.